packages/stylelint-config/README.md

```diff
@@ -19,27 +19,113 @@ npm install --save-dev @lmc-eu/stylelint-config stylelint-config-prettier
 > We assume you are using Prettier. That's why we also recommend adding
 > `stylelint-config-prettier` above.
 
-## Configurations
+New configuration extends community maintained config [stylelint-config-standard-scss](https://github.com/stylelint-scss/stylelint-config-standard-scss).
+
+This config extends another community maintained configs:
+
+- [stylelint-config-recommended](https://github.com/stylelint/stylelint-config-recommended)
+- [stylelint-config-recommended-scss](https://github.com/stylelint-scss/stylelint-config-recommended-scss)
+- [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard)
+
+## Configuration
 
 ### `@lmc-eu/stylelint-config`
 
 Use this ruleset to configure Stylelint to work with your code.
 
-## Optional configurations
+## Recommended stylelint config
 
-### `@lmc-eu/stylelint-config/optional`
+```js
+// .stylelintrc.js
 
-Use this ruleset in conjunction with any of the above rulesets. Provides additional insights into potential inconsistencies in the project.
+'use strict';
 
-## Recommended stylelint config
+module.exports = {
+  extends: ['@lmc-eu/stylelint-config', 'stylelint-config-prettier'],
+};
+```
+
+<details>
+<summary><i>.stylelintrc</i> or <i>.stylelintrc.json</i></summary>
+
+```json
+{
+  "extends": ["@lmc-eu/stylelint-config", "stylelint-config-prettier"]
+}
+```
+
+</details>
+
+<details>
+<summary><i>package.json</i></summary>
+
+```json
+{
+  "stylelint": {
+    "extends": ["@lmc-eu/stylelint-config", "stylelint-config-prettier"]
+  }
+}
+```
+
+</details>
+
+---
+
+## Legacy Configurations
+
+Use this config to keep less strict ruleset that consists from the original one up to this version:
+
+`@lmc-eu/stylelint-config/legacy`
+
+or its extended version by rules for properties order (formerly known as `optional` until now):
```

Comment:
And again the version number to make it future proof:

```suggestion
or its extended version by rules for properties order (formerly known as `optional` until version 5.x):
```

Reply:
Fixed but this you may be misunderstood - this line still explains old config but also extended by properties order. This line should still point to 4.0.1 version, the old one.